cpp/src/lance/format/schema.cc

```diff
@@ -702,4 +702,24 @@ pb::Field::Type Field::GetNodeType() const {
   }
 }
 
+void Print(const Field& field, const std::string& path, int indent = 0) {
```

Comment:
would it make sense to print the storage type for extension types here as well?

Reply:
Sure can do